### PR TITLE
fix: limit slippage

### PR DESCRIPTION
**Pull request by nick134-bit**
## Related Issues

---

## Checklist:

- [ ] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-frontend/blob/main/CONTRIBUTING.md).
- [ ] My pull request has a sound title and description (not something vague like `Update index.md`)
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation.
- [ ] The code is formatted properly `yarn lint`.
- [ ] The project builds and is able to deploy on Netlify `yarn build`.

### CODE REVIEW

File: components/Pages/Trade/Liquidity/hooks/useDepositTransaction.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import useDebounceValue from 'hooks/useDebounceValue'
 import { executeAddLiquidity } from 'services/liquidity/index'
 import { TxStep } from 'types/common'
 
+import { getPoolInfo } from '../../../../../services/swap'
```
Comment (worrex) [inline components/Pages/Trade/Liquidity/hooks/useDepositTransaction.tsx:11]: import kürzen

File: components/Pages/Trade/Liquidity/hooks/useDepositTransaction.tsx [hunk 2]
```diff
@@ -49,9 +51,29 @@ export const useTransaction = ({
...
+  const confirmRatio = async (debouncedMsgFunds:Array<any>) => {
+    const address = msgs.provide_liquidity ? swapAddress : msgs.deposit.pair_address;
+    const poolInfo = await getPoolInfo(address, signingClient);
+    const poolAssets = poolInfo.assets;
+
+    const [poolAsset1, poolAsset2] = poolAssets;
+    const poolRatio = poolAsset1.amount / poolAsset2.amount;
+
+    const transactionRatio =
+      Number(debouncedMsgFunds.find((elem: any) => elem.denom === poolAsset1.info.native_token.denom).amount) /
```
Comment (worrex) [inline components/Pages/Trade/Liquidity/hooks/useDepositTransaction.tsx:63]: considered cw20 ? -> ...info.contract_address.denom glaube ists bei cw20

Comment (nick134-bit) [reply components/Pages/Trade/Liquidity/hooks/useDepositTransaction.tsx:63]: K I check again with a cw20 token

**Review by worrex — CHANGES_REQUESTED** [review]
siehe comments

**Review by 0xFable — APPROVED** [review]

**Pull request merged**